gitpod-io/gitpod
# WIP: [dashboard] display warning for latest IDE versions

## Description

display warning for latest IDE versions on the loading screen for latest versions like VS Code Insiders and EAP of JB products

- [ ] Add test cases
- [ ] Need design (message detail, display position, adjust if content height too long

~~I can split commits into `format` and `code` if you want. done~~

## Related Issue(s)

Fixes #8028 

## How to test

1. Switch your Preferences
2. Open workspace, see if warning display
<img width="726" alt="image" src="https://user-images.githubusercontent.com/20944364/158101112-450e7112-2a52-4ed4-b32a-ec00ea5026e6.png">

## Release Notes

```release-note
Display warning message when users choose latest IDE
```

## Documentation
